cloud/pkg/cloudstream/streamserver.go

```diff
@@ -153,18 +153,11 @@ func (s *StreamServer) getMetrics(r *restful.Request, w *restful.Response) {
 		return
 	}
 
-	w.Header().Set("Transfer-Encoding", "chunked")
 	w.WriteHeader(http.StatusOK)
```

Comment:
need statusok？

Reply:
yes，kubelet actually call the writeAsJson function，which passes the status_ok parameter